code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -109,6 +111,8 @@
 	reagent_tag = PROCESS_ORG
 	flesh_color = "#AFA59E"
 	base_color = "#333333"
+	//Default styles for created mobs.
+	default_headacc = "Tajaran Ears"
```

Comment:
Means that, for example, when creating a mob from the admin game panel, they will spawn with this head accessory style.